### DISCUSSION TITLE
Can we have some clarifications around Windows node access?

### DISCUSSION THREAD

**Discussion by Raptus1** [netdata/netdata#18422] [Q&A]
From the latest release notes about the upcoming 2.0 release:

 "However, once mark this as GA, access to Windows nodes will be restricted to customers with a Netdata paid plan."

It is not quite clear what this means, will the metrics not get streamed to a parent from a Windows based Netdata Agent or is the local dashboard simply unavailable?

Please clarify this here. We are very fond of the initial netdata project, but now it seems like more and more pay for play is being pushed hinting to a negative trend in how Netdata ecosystems is evolving.

**Comment by esimeon-hc** [comment]
I was looking forward to the release of the Windows Agent.
I need to deploy the Windows Agent on servers that are behind the firewall and not accessible to/from the Internet.
I was planning on disabling the GUI and connect the agent to Netdata parent instance.

Is this default configuration for Unix and other environment, available/supported for the Windows Agent?

Thank you,
NetData is a great product.
I am coming from Prometheus and was hoping to migrate over.

**Reply by ilyam8** [reply]
Hi, @esimeon-hc . The answer is yes. We recommend using a Linux host as a Parent (better performance).

**Reply by esimeon-hc** [reply]
@ilyam8 
I didn't mean to hijack @Raptus1's note.
Please comment on the subscription question:

"However, once mark this as GA, access to Windows nodes will be restricted to customers with a Netdata paid plan."

**Reply by ilyam8** [reply]
@Raptus1 

> will the metrics not get streamed to a parent from a Windows

Will. But Windows nodes will be UI-locked on free plans.

**Comment by esimeon-hc** [comment]
Thank you for the clarification.
Windows nodes will be UI-locked on free plans.

But, will I still be able to access the metrics via an API endpoint,

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Raptus1** [comment]
Thansk for the reply :=

What do you mean with UI-locked on free plan..?
Is it possible to add to a non-cloud parent? Looking at the [installer docs](https://learn.netdata.cloud/docs/netdata-agent/installation/windows#silent-mode-command-line), it is only configurable with proxy and or Cloud token.

Will there be a netdata.conf or stream.conf to use?
